src/Form/test-cases/esty.js

```diff
@@ -0,0 +1,1064 @@
+module.exports = `
+<form method="POST" class="add-new-form" id="cc-panel--paymentstep">
```

Comment:
The form is not super well formatted. I have just copied from the devtools and pasted here. If I ran it through a beautifier, it became much longer (mainly because of the svgs). I even considered "cleaning" it up by removing some markup, like the svgs, but I don't want this to be the expectations. Actually, I want whoever adds these files to do it as fast as possible by just dumping whatever comes from the devtools, applying the attributes we need and move on. We don't need them to be pretty.